makefile

```diff
@@ -37,6 +37,9 @@ run-local-db:
 	@echo "Running local DB container"
 	@docker-compose -f docker-compose.dev.yml up mongodb
 
+run-e2e-test:
```

Comment:
Should we use this in `.github/workflows/test.yml`? How does this differ from `make local-server-tests`?

Reply:
I just made this to run the tests locally and not having to cd server every time and load env vars manually.